test/unit/io/sequence_file/sequence_file_format_embl_test.cpp

```diff
@@ -357,6 +379,29 @@ TEST_F(read, seq_qual)
     }
 }
 
+TEST_F(read, illegal_alphabet)
+{
+    std::string input
+    {
+        "IK ID1;\tstuff\n"
```

Comment:
This test throws already for the wrong ID identifier, right? So illegal alphabet characters are not actually tested...